interview-questions.md

```diff
@@ -8,48 +8,52 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is JSX? What is one syntax difference between HTML and JSX?
 
-Your answer:
+Your answer: JSX is a markup extension to Javascript. It is similar to HTMl, but it is not as JSX is for React and was created by React. 
 
-Researched answer:
+Researched answer: JSX is a markdown syntax extension to Javascript and it is combining the logic and the markup together. One syntax difference between the two are the empty fragments and also, you must return every parent element. Tags can also self close such as a component call, unlike HTML everything needs an opening and closing tag. 
+
+<!-- Sources from https://www.freecodecamp.org/news/html-vs-jsx-whats-the-difference/#:~:text=HTML%20is%20a%20very%20important,a%20syntactic%20sugar%20for%20React. -->
 
 2. What is yarn? What file(s) are modified in a React application when you run yarn?
 
-Your answer:
+Your answer: Yarn stands for Yet Another Resource Negotiator and is the file manager for Javascript. I believe node modules are modified. 
 
-Researched answer:
+Researched answer: I'm not exactly sure what the question means, but when yarn is added there a bunch of files that are added such as the node modules which are all the dependencies that are needed for React App. 
```

Comment:
Running yarn does add node modules and will also update package.json and yarn.lock if any new dependencies have been added.